devtools/create_achievements/steam_achievements.py

```diff
@@ -136,10 +163,10 @@ def parse_steamdb_stats(url):
 	for i in range(achievements_entries):
 		idx   = achievements_columns * i
 		name  = achievements_rows[idx + 0].text.strip()
-		texts = achievements_rows[idx + 1].text.strip().split("\n")
+		texts = stringify(achievements_rows[idx + 1]).strip().replace("\n\n", "\n").split("\n")
```

Comment:
some rows include a <p> to render the description text in italic. but using stringify() you thus get

```
title\n
\n
description\n
```